vertical-tabbrowser.xml

```diff
@@ -450,37 +484,38 @@
             // also avoid detaching if the the tab was dropped too close to
             // the tabbar (half a tab)
             let endScreenY = bo.screenY + 1.5 * bo.height;
-            if (eY < endScreenY && eY > window.screenY)
+            if (eY < endScreenY && eY > window.screenY) {
               return;
+            }
           }
         }
 
         // screen.availLeft et. al. only check the screen that this window is on,
         // but we want to look at the screen the tab is being dropped onto.
-        var sX = {}, sY = {}, sWidth = {}, sHeight = {};
-        Cc["@mozilla.org/gfx/screenmanager;1"]
+        let sX = {}, sY = {}, sWidth = {}, sHeight = {};
+        Cc['@mozilla.org/gfx/screenmanager;1']
           .getService(Ci.nsIScreenManager)
           .screenForRect(eX, eY, 1, 1)
           .GetAvailRect(sX, sY, sWidth, sHeight);
         // ensure new window entirely within screen
-        var winWidth = Math.min(window.outerWidth, sWidth.value);
-        var winHeight = Math.min(window.outerHeight, sHeight.value);
-        var left = Math.min(Math.max(eX - draggedTab._dragData.offsetX, sX.value),
+        let winWidth = Math.min(window.outerWidth, sWidth.value);
+        let winHeight = Math.min(window.outerHeight, sHeight.value);
+        let left = Math.min(Math.max(eX - draggedTab._dragData.offsetX, sX.value),
                             sX.value + sWidth.value - winWidth);
-        var top = Math.min(Math.max(eY - draggedTab._dragData.offsetY, sY.value),
+        let top = Math.min(Math.max(eY - draggedTab._dragData.offsetY, sY.value),
                            sY.value + sHeight.value - winHeight);
 
         delete draggedTab._dragData;
 
-        if (this.tabbrowser.tabs.length == 1) {
+        if (this.tabbrowser.tabs.length === 1) {
           // resize _before_ move to ensure the window fits the new screen.  if
           // the window is too large for its screen, the window manager may do
           // automatic repositioning.
           window.resizeTo(winWidth, winHeight);
           window.moveTo(left, top);
           window.focus();
         } else {
-          this.tabbrowser.replaceTabWithWindow(draggedTab, { screenX: left,
+          this.tabbrowser.replaceTabWithWindow(draggedTab, {screenX: left,
                                                              screenY: top,
```

Comment:
I think we should probably make `screenY` line up with the new position of `screenX`…